GW2EIEvtcParser/EncounterLogic/Raids/W8/UraTheSteamshrieker.cs

```diff
@@ -21,7 +21,9 @@ public UraTheSteamshrieker(int triggerID) : base(triggerID)
     {
         MechanicList.AddRange(new List<Mechanic>()
         {
-            new PlayerDstHitMechanic(EruptionVent, new MechanicPlotlySetting(Symbols.TriangleSW, Colors.DarkPink), "ErupVent.H", "Hit by Eruption Vent (Geyser Spawn)", "Eruption Vent Hit", 0),
+            new PlayerDstHitMechanic(EruptionVent, new MechanicPlotlySetting(Symbols.TriangleSW, Colors.DarkPink), "ErupVent.H", "Hit by Eruption Vent (Geyser Shockwave)", "Eruption Vent Hit", 0),
+            new PlayerDstHitMechanic(EruptionVent, new MechanicPlotlySetting(Symbols.TriangleSW, Colors.Pink), "Achiv.Hop", "Achievement Eligibility: Hopscotch Master", "Achiv: Hopscotch Master", 0)
+                .UsingEnable(log => log.FightData.IsCM).UsingAchievementEligibility(true),
```

Comment:
log.FightData.IsCM is enough for legendary mode too, correct?

Reply:
no, IsLCM flag needs to be used